{"total_count": 2, "incomplete_results": false, "items": [{"url": "https://api.github.com/repos/DariuszPorowski/azure-pipelines-tasks-mysql/issues/2", "repository_url": "https://api.github.com/repos/DariuszPorowski/azure-pipelines-tasks-mysql", "labels_url": "https://api.github.com/repos/DariuszPorowski/azure-pipelines-tasks-mysql/issues/2/labels{/name}", "comments_url": "https://api.github.com/repos/DariuszPorowski/azure-pipelines-tasks-mysql/issues/2/comments", "events_url": "https://api.github.com/repos/DariuszPorowski/azure-pipelines-tasks-mysql/issues/2/events", "html_url": "https://github.com/DariuszPorowski/azure-pipelines-tasks-mysql/issues/2", "id": 233408963, "node_id": "MDU6SXNzdWUyMzM0MDg5NjM=", "number": 2, "title": "Error running MySQL script: Exception calling \"Fill\" with \"1\" argument(s): \"Index was outside the bounds of the array.\"", "user": {"login": "jawandarajbir", "id": 3058693, "node_id": "MDQ6VXNlcjMwNTg2OTM=", "avatar_url": "https://avatars.githubusercontent.com/u/3058693?v=4", "gravatar_id": "", "url": "https://api.github.com/users/jawandarajbir", "html_url": "https://github.com/jawandarajbir", "followers_url": "https://api.github.com/users/jawandarajbir/followers", "following_url": "https://api.github.com/users/jawandarajbir/following{/other_user}", "gists_url": "https://api.github.com/users/jawandarajbir/gists{/gist_id}", "starred_url": "https://api.github.com/users/jawandarajbir/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/jawandarajbir/subscriptions", "organizations_url": "https://api.github.com/users/jawandarajbir/orgs", "repos_url": "https://api.github.com/users/jawandarajbir/repos", "events_url": "https://api.github.com/users/jawandarajbir/events{/privacy}", "received_events_url": "https://api.github.com/users/jawandarajbir/received_events", "type": "User", "site_admin": false}, "labels": [{"id": 527155586, "node_id": "MDU6TGFiZWw1MjcxNTU1ODY=", "url": "https://api.github.com/repos/DariuszPorowski/azure-pipelines-tasks-mysql/labels/bug", "name": "bug", "color": "ee0701", "default": true, "description": null}], "state": "closed", "locked": true, "assignee": {"login": "DariuszPorowski", "id": 3431813, "node_id": "MDQ6VXNlcjM0MzE4MTM=", "avatar_url": "https://avatars.githubusercontent.com/u/3431813?v=4", "gravatar_id": "", "url": "https://api.github.com/users/DariuszPorowski", "html_url": "https://github.com/DariuszPorowski", "followers_url": "https://api.github.com/users/DariuszPorowski/followers", "following_url": "https://api.github.com/users/DariuszPorowski/following{/other_user}", "gists_url": "https://api.github.com/users/DariuszPorowski/gists{/gist_id}", "starred_url": "https://api.github.com/users/DariuszPorowski/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/DariuszPorowski/subscriptions", "organizations_url": "https://api.github.com/users/DariuszPorowski/orgs", "repos_url": "https://api.github.com/users/DariuszPorowski/repos", "events_url": "https://api.github.com/users/DariuszPorowski/events{/privacy}", "received_events_url": "https://api.github.com/users/DariuszPorowski/received_events", "type": "User", "site_admin": false}, "assignees": [{"login": "DariuszPorowski", "id": 3431813, "node_id": "MDQ6VXNlcjM0MzE4MTM=", "avatar_url": "https://avatars.githubusercontent.com/u/3431813?v=4", "gravatar_id": "", "url": "https://api.github.com/users/DariuszPorowski", "html_url": "https://github.com/DariuszPorowski", "followers_url": "https://api.github.com/users/DariuszPorowski/followers", "following_url": "https://api.github.com/users/DariuszPorowski/following{/other_user}", "gists_url": "https://api.github.com/users/DariuszPorowski/gists{/gist_id}", "starred_url": "https://api.github.com/users/DariuszPorowski/starred{/owner}{/repo}", "subscriptions_url": "https://api.github.com/users/DariuszPorowski/subscriptions", "organizations_url": "https://api.github.com/users/DariuszPorowski/orgs", "repos_url": "https://api.github.com/users/DariuszPorowski/repos", "events_url": "https://api.github.com/users/DariuszPorowski/events{/privacy}", "received_events_url": "https://api.github.com/users/DariuszPorowski/received_events", "type": "User", "site_admin": false}], "milestone": null, "comments": 3, "created_at": "2017-06-04T03:16:40Z", "updated_at": "2018-01-28T23:03:43Z", "closed_at": "2018-01-28T23:03:43Z", "author_association": "NONE", "active_lock_reason": null, "body": "Hi Dariusz,\r\n\r\nI'm trying to run a MySQL script against a ClearDB hosted MySQL database. This script is running just fine using MySQL workbench on same target DB and using exact same credentials. However, when executed on Visual Studio Online build using \"Run MySQL script\" task, I'm seeing following error. \r\n\r\n`Error running MySQL script: Exception calling \"Fill\" with \"1\" argument(s): \"Index was outside the bounds of the array.\"`\r\n![mysql error](https://cloud.githubusercontent.com/assets/3058693/26758645/d09ffe32-4927-11e7-8cc3-5a876152eaed.png)\r\n\r\nAppreciate your help here :)\r\n\r\n", "reactions": {"url": "https://api.github.com/repos/DariuszPorowski/azure-pipelines-tasks-mysql/issues/2/reactions", "total_count": 0, "+1": 0, "-1": 0, "laugh": 0, "hooray": 0, "confused": 0, "heart": 0, "rocket": 0, "eyes": 0}, "timeline_url": "https://api.github.com/repos/DariuszPorowski/azure-pipelines-tasks-mysql/issues/2/timeline", "performed_via_github_app": null, "state_reason": "completed", "score": 1.0}]}